README.md

```diff
@@ -49,29 +49,103 @@ polygon detection is performed and checked if the the number of sides is more th
 * This is not robust enough, the thresholds need to be adjusted always.
 * Doesnt work properly on real world data as there is lot of noise. 
 
-### Real World (Test Lot) --- YOLOv3-tiny (You Only Look Once)
-We used this approach for real world.
-TODO:write about it
-
-### Real World (Test Lot) --- SSD (Single Shot Detection)
+## SSD Approach (Single Shot Detection)
```

Comment:
I just noticed that SSD uses `##` instead of `###`. Since it falls under "Traffic Light Detection Node" section, which has `##`, SSD should have `###`. The same applies to other paragraphs under the SSD section. `###` should be replaced with `####` and `####` should be replaced with `#####`.

Reply:
Corrected